lib/tools/system-calls.js

```diff
@@ -344,6 +345,23 @@ systemCallMethods.killAllEmulators = async function () {
   }
 };
 
+systemCallMethods.killEmulator = async function (avdName) {
+  let cmd;
+  if (system.isWindows()) {
+    cmd = `for /f "tokens=2 delims=," %a in ('tasklist /FI "imagename eq emulator*" /V /FO:csv /NH ^| findstr /r /c:".*:${avdName}.*"') do taskkill /pid %a /F`;
+  } else {
+    cmd = `pkill -9 -f 'avd ${avdName}|@${avdName}'`;
+  }
+  try {
+    log.debug(`killing avd '${avdName}'`);
+    await (B.promisify(cp.exec))(cmd);
+    await sleep(1000);
```

Comment:
Is there nothing that could be watched for the actual effect? Logcat, or something?